docs/openstack/backend_services_deployment.md

```diff
@@ -218,6 +272,7 @@ podified OpenStack control plane services.
               metadata:
                 annotations:
                   metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
```

Comment:
the same

Reply:
this works for my testing w/o the allow-shared-ip label. That looks correct as we are not going to share those VIPs